Sming/Components/Storage/src/include/Storage/PartitionTable.h

```diff
@@ -45,7 +45,7 @@ class PartitionTable
 
 	/**
 	 * @brief Find partition by name
-	 * @param Name to search for, case-sensitive
+	 * @param Name Name to search for, case-sensitive
```

Comment:
`Name Name to search` should be `Name to search`.

Reply:
Here's what appears in the documentation:

![image](https://user-images.githubusercontent.com/1982774/111330762-24612880-8668-11eb-80c5-45d4235d9b28.png)

Reply:
> Here's what appears in the documentation:

Thank for clarifying. Ignore my comment.